assets/locales/en.json

```diff
@@ -74,7 +87,8 @@
         "logoText":"Ocean Pearl",
         "navbarLinkDefault":"Home",
         "navbarLink1":"Projects",
-        "navbarLink2":"Jobs"
+        "navbarLink2":"Dao",
+        "navbarLink3":"Jobs"
```

Comment:
suggestion (non-blocking): Maybe we want to change the order later. It's probably better to use another naming convention like `navbarJobs, navbarDao, navbarJobs`